app/src/main/java/chat/rocket/android/helper/AnimationHelper.kt

```diff
@@ -0,0 +1,52 @@
+package chat.rocket.android.helper
+
+import android.animation.Animator
+import android.animation.AnimatorListenerAdapter
+import android.animation.AnimatorSet
+import android.animation.ObjectAnimator
+import android.content.Context
+import android.os.Build
+import android.os.VibrationEffect
+import android.os.Vibrator
+import android.view.View
+
+object AnimationHelper {
```

Comment:
Again, lets avoid Helper classes and use extensions...

Reply:
I don't like populating our code with a lot of extensions functions. This class can easily decompiled to a java version, allowing future uses in non-Kotlin projects (i.e Java). Also, if the idea is to create lots files to keep the extensions functions, what is the diference of having classes like that? Unless we have a big file to keep our extensions (that wouldn't be good).

Always recommended: https://www.philosophicalhacker.com/post/how-to-abuse-kotlin-extension-functions/